components/http-client/src/api_client.rs

```diff
@@ -1,82 +1,96 @@
-use std::{path::Path,
+use std::{fs::{self,
+               File},
+          io::{Read,
+               Write},
+          iter::FromIterator,
+          path::{Path,
+                 PathBuf},
+          str::FromStr,
           time::Duration};
 
+use reqwest::{header::{HeaderMap,
+                       HeaderValue,
+                       USER_AGENT},
+              Certificate,
+              Client as ReqwestClient,
+              ClientBuilder,
+              IntoUrl,
+              Proxy,
+              RequestBuilder};
+
 use habitat_core::{env,
-                   package::PackageTarget,
+                   fs::cache_ssl_path,
+                   package::{PackageIdent,
+                             PackageInstall,
+                             PackageTarget},
                    util::sys};
-use hyper::{client::{pool::{Config,
-                            Pool},
-                     Client as HyperClient,
-                     IntoUrl,
-                     RequestBuilder},
-            header::UserAgent,
-            http::h1::Http11Protocol,
-            net::HttpsConnector};
-use hyper_openssl::OpensslClient;
-use openssl::ssl::{SslConnector,
-                   SslConnectorBuilder,
-                   SslMethod,
-                   SslOption,
-                   SSL_OP_NO_COMPRESSION,
-                   SSL_OP_NO_SSLV2,
-                   SSL_OP_NO_SSLV3,
-                   SSL_VERIFY_NONE};
+
 use url::Url;
 
-use crate::{error::{Error,
-                    Result},
-            net::ProxyHttpsConnector,
-            proxy::{proxy_unless_domain_exempted,
-                    ProxyInfo},
-            ssl};
```

Comment:
Same comment as previously re: deleting blank lines in the `use` statements.

Reply:
OK